src/mlpack/core/util/cli_impl.hpp

```diff
@@ -23,16 +23,19 @@ namespace mlpack {
  * @tparam T The type of the parameter.
  * @param identifier The name of the parameter, eg foo in bar/foo.
  * @param description A string description of the parameter.
- * @param parent The name of the parent of the parameter,
- *   eg bar/foo in bar/foo/buzz.
- * @param required If required, the program will refuse to run
- *   unless the parameter is specified.
+ * @param parent The name of the parent of the parameter, e.g. bar/foo in
+ *     bar/foo/buzz.
+ * @param required If required, the program will refuse to run unless the
+ *     parameter is specified.
+ * @param input If true, the parameter is an input parameter (not an output
+ *     parameter).
  */
 template<typename T>
 void CLI::Add(const std::string& identifier,
               const std::string& description,
               const std::string& alias,
```

Comment:
It's some inconsistency from the old code, but I think we should use `parent` here.